install/installation-kubernetes.md

```diff
@@ -0,0 +1,174 @@
+1.  Add the TimescaleDB Helm chart repository:
+    ```bash
+    helm repo add timescale 'https://charts.timescale.com'
+    ```
+1.  Verify that the repository is up to date:
+    ```bash
+    helm repo update
+    ```
+1.  Install the TimescaleDB Helm chart, by replacing `<MY_NAME>` with a name of
+    your choice:
+    ```bash
+    helm install <MY_NAME> timescale/timescaledb-single
+    ```
+    If you created a `<MY_VALUES.yaml>` file, use this command instead:
+    ```bash
+    helm install <MY_NAME> -f <MY_VALUES.yaml> charts/timescaledb-single
+    ```
+
+</procedure> 
+
+## Connect to TimescaleDB
+You can connect to TimescaleDB from an external IP address, or from within the
+cluster.
+
+<procedure>
+
+### Connecting to TimescaleDB using an external IP
+
+<highlight type="note">
+If you configured the user credentials in the `my_values.yaml` file, you don't
+need to decode the passwords. In the following section replace `<MY_NAME>` with
+the name that you provided during the installation.
+</highlight>
+
+1. Get the name of the host to connect to:
+    ```bash
+    kubectl get service/<MY_NAME>
+    ```
+1. Decode the `admin` user password `PGPASSWORD_ADMIN` that was generated during
+   the Helm installation:
+    ```bash
+    PGPASSWORD_ADMIN=$(kubectl get secret --namespace default 
+    <MY_NAME>-credentials -o jsonpath="{.data.PATRONI_admin_PASSWORD}" | base64 --decode)
+    ``` 
+1. **OPTIONAL** Decode the super user password `PGPOSTGRESPASSWORD` that was
+   generated during the Helm installation:
+    ```bash
+    PGPASSWORD_POSTGRES=$(kubectl get secret --namespace default 
+    <MY_NAME>-credentials -o jsonpath="{.data.PATRONI_SUPERUSER_PASSWORD}" | base64 --decode)
+    ```
+1. Connect with psql as `admin` user:
+    ```bash
+    kubectl run -i --tty --rm psql --image=postgres \
+      --env "PGPASSWORD=$PGPASSWORD_ADMIN" \
+      --command -- psql -U admin \
+      -h <MY_NAME>.default.svc.cluster.local postgres
+    ```
+    
+</procedure>
+
+
+### Connecting to TimescaleDB from inside the cluster
+1. Get the Pod on which TimescaleDB is installed:
+   ```bash
+    MASTERPOD="$(kubectl get pod -o name --namespace default -l release=test,role=master)"
+    ```
+1. Run `psql` inside the Pod containing the primary:
+    ```bash
+    kubectl exec -i --tty --namespace default ${MASTERPOD} -- psql -U postgres
+    ```
+
+</procedure>
+
+## Create a database
+ After installing and connecting to TimescaleDB you can create a database,
+ connect to the database, and also verify that the TimescaleDB extension is
+ installed.
+ 
+<procedure>
+
+### Creating a database
+1.  At the prompt, create an empty database. For example, to create a database
+    called `tsdb`:
+    ```sql
+    CREATE database tsdb;
+    ```
+1.  Connect to the database you created:
+    ```sql
+    \c tsdb
+    ```
+1. Verify that the TimescaleDB extension is installed by using the `\dx`
+command at the command prompt. It looks like this:
```

Comment:
I think the indentation here is breaking the procedure block. L124 needs to be indented:

![image](https://user-images.githubusercontent.com/3914967/161905496-7fbc84de-4e69-43b2-820f-dccfbb93da48.png)

Reply:
Hi @Loquacity !!! Yes, I had fixed this in the [commit](https://github.com/timescale/docs/pull/911/commits/d71742caef29aaa9ec7d5cef4a31df92968da6aa)

Reply:
It doesn't look like that commit touched L124 ... ?

Reply:
Thanks @Loquacity !!! You have an eye of a hawk. My bad I missed it. Just fixed the indentation.